.github/workflows/continuous-integration-workflow.yml

```diff
@@ -26,44 +26,52 @@ jobs:
         cmake_build_type: ['Release', 'Debug']
         backend: ['OPENMP']
         clang-tidy: ['']
+        stdcxx: [17]
```

Comment:
Given how all the jobs overwrite all the parameters in the `matrix`, should those parameters all be empty? That would make sure that if a job does not overwrite a parameter, it won't run with the full range.

Reply:
Most of the jobs actually use those values. We only have a couple of specialized extra builds that use custom parameters.